### PR TITLE
Add Child Care Coordinator instructions and use percentage for attendance estimate

### PR DESCRIPTION
**Pull request by mxie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,9 +101,19 @@ Find an MC.
...
 ## 1.5 weeks before (Catering Coordinator)
 
-We place catering orders ~1.5 weeks before the workshop. We assume about 100
-people will attend, but double check that number. For more information, see
-[this document](/catering.md).
+We place catering orders ~1.5 weeks before the workshop. We assume about 75% of
+the people who registered + number of TA's will attend, but double check that
+number. For more information, see [this document](/catering.md).
+
+## 1 weeks before (Child Care Coordinator)
```
Comment (gabebw) [inline README.md]: week (singular)

Comment (gabebw) [inline README.md]: How about noting at the top that you're the Child Care Coordinator?

**Comment by gabebw** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (mxie) [inline README.md]: Yeah sure!

**Pull request merged**